clients/csharp-uwp/UWPVoiceAssistantSample/DirectLineSpeechDialogBackend.cs

```diff
@@ -242,7 +265,7 @@ public async Task StartAudioTurnAsync(bool performConfirmation)
         public async Task CancelSignalVerification()

```

Comment:
Can we make this `CancelSignalVerificationAsync` to be consistent with the method naming patterns? (I know, not this change, but still 😁 )

Reply:
Agreed, changed